service/rest_patch.go

```diff
@@ -62,6 +64,20 @@ func (restapi restAPI) getPatchConfig(w http.ResponseWriter, r *http.Request) {
 	}
 	w.Header().Set("Content-Type", "application/x-yaml; charset=utf-8")
 	w.WriteHeader(http.StatusOK)
-	_, err := w.Write([]byte(projCtx.Patch.PatchedParserProject))
+	var err error
+	if projCtx.Patch.PatchedParserProject != "" {
+		_, err = w.Write([]byte(projCtx.Patch.PatchedParserProject))
+	} else {
+		var pp *model.ParserProject
```

Comment:
This line can be removed

Reply:
We need it to then return the error correctly at 82 (although i agree it's not clear so i'll restructure)